{{cookiecutter.app_name}}/helpers/config.go

```diff
@@ -0,0 +1,30 @@
+package helpers
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/spf13/viper"
+)
+
+func GetConfigPrefix() string {
```

Comment:
Since this method is only used internally, how about not exporting it?

Reply:
I export this function because I want to write unit test. `config_test`'s package is `helpers_test`.

Reply:
I saw that but I am not sure of the value-added to have such test as it will be implicitly tested in the other public methods.